__mocks__/@expensify/react-native-live-markdown.ts

```diff
@@ -0,0 +1,14 @@
+import {MarkdownTextInput} from '@expensify/react-native-live-markdown';
+import type {parseExpensiMark} from '@expensify/react-native-live-markdown';
+
+global.jsi_setMarkdownRuntime = jest.fn();
+global.jsi_registerMarkdownWorklet = jest.fn();
+global.jsi_unregisterMarkdownWorklet = jest.fn();
+
+const parseExpensiMarkMock: typeof parseExpensiMark = () => {
+    'worklet';
+
+    return [];
+};
+
+export {MarkdownTextInput, parseExpensiMarkMock as parseExpensiMark};
```

Comment:
Is it okay if we have this mock here or should we move it to `react-native-live-markdown` somehow?

Reply:
This part of the mock can be moved. But I'm not sure about the react-native part. Maybe we can just avoid [throwing an error if NativeLiveMarkdownModule is not available](https://github.com/Expensify/react-native-live-markdown/blob/540db5fd3103d8239a8e0a0b8d8bcb71e28913eb/src/MarkdownTextInput.tsx#L31-L33)? (same as it was before). The global.jsi_setMarkdownRuntime check is probably enough

Reply:
Raised PR https://github.com/Expensify/react-native-live-markdown/pull/578.

Reply:
Thanks for the PR! Left couple of comments there.

Reply:
Yes, let's wait for the bump. I've asked my collegaues to review the PR with the mock (it looks good for me, just wanted another pair of eyes on it). I hope we can merge that PR today. Then we can bump here and ask for QA.

Reply:
@s77rt I've just merged the PR introducing the mock to `react-native-live-markdown` (https://github.com/Expensify/react-native-live-markdown/pull/578). Let's bump live-markdown in this PR, remove mock here and we should be ready to test.

Reply:
Done! This is ready for test